src/components/IconBar/IconBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        height: '100%',
+        margin: 0,
+    },
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        minHeight: '100vh',
+    },
+    footer: {
+        backgroundColor: theme.palette.background.paper,
+        textAlign: 'center',
+        padding: '1em',
+    },
+}));
+
+interface Props {
+    children: JSX.Element | JSX.Element[];
+    justify: string;
+}
+
+function convertNum(children: number): 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 {
+    return Math.round(12 / children) as
+        | 1
+        | 2
+        | 3
+        | 4
+        | 5
+        | 6
+        | 7
+        | 8
+        | 9
+        | 10
+        | 11
+        | 12;
+}
```

Comment:
I think there may be an issue here.  If, for example, we have 100 children, then 12/100 is .12 which gets rounded to 0 -- which is not any number between 1-12.  I may have said to use Math.round when Math.ceil is better.  It may not be a realistic example, but we should make everything airtight.  Did `auto` not work as expected?

Reply:
auto did not work, xs only accepts numeric literal types 1 - 12, and Math.round is type number

Reply:
I was under the impression that we wanted all the children on one row, so thats why I used Math.round, if we dont mind using multiple rows I can use Math.ceil

Reply:
Yeah, lets use multiple rows if necessary.  So I think Math.ceil is better in this case.